### DISCUSSION TITLE
[@types/react] Issue when updating form 18.0.1 to 18.0.2

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [DefinitelyTyped/DefinitelyTyped#59843] [Issues with a @types package]
This is the branch that is failing.

https://github.com/The-Code-Monkey/TechStack/pull/175

I only have one version in my yarn.lock as that issue was fixed when updating to 18.0.1

not sure what the issue actually is here, as its erroring saying i dont have it installed.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
This issue is tracked in https://github.com/vercel/next.js/issues/36085
